src/app/forms/controls/radio.html

```diff
@@ -0,0 +1,179 @@
+              <div class="clr-radio-wrapper">
+                  <input type="radio" id="basic" class="clr-radio">
+                  <label for="basic">Label with for</label>
+              </div>
+              <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+              <span class="clr-subtext">Helper Text</span>
+          </div>
+      </div>
+  
+      <div class="clr-form-control">
+          <label class="clr-control-label">
+              Multi
+          </label>
+          <div class="clr-control-container">
+              <div class="clr-radio-wrapper">
+                  <input type="radio" id="multi-1" class="clr-radio">
+                  <label for="multi-1" class="clr-control-label">Label with for</label>
+                  <input type="radio" id="multi-2" class="clr-radio">
+                  <label for="multi-2" class="clr-control-label">Label with for</label>
+                  <input type="radio" id="multi-3" class="clr-radio">
+                  <label for="multi-3" class="clr-control-label">Label with for</label>
+                  <input type="radio" id="multi-1" class="clr-radio">
+                  <label for="multi-1" class="clr-control-label">Label with for</label>
+                  <input type="radio" id="multi-2" class="clr-radio">
+                  <label for="multi-2" class="clr-control-label">Label with for</label>
+                  <input type="radio" id="multi-3" class="clr-radio">
+                  <label for="multi-3" class="clr-control-label">Label with for</label>
+              </div>
+              <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+              <span class="clr-subtext">Helper Text</span>
+          </div>
+      </div>
+  
+      <div class="clr-form-control">
+          <label class="clr-control-label">
+              Multi inline
+          </label>
+          <div class="clr-control-container">
+              <div class="clr-radio-wrapper clr-radio-inline">
+                  <input type="radio" id="multi-1" class="clr-radio">
+                  <label for="multi-1" class="clr-control-label">Label with for</label>
+                  <input type="radio" id="multi-2" class="clr-radio">
+                  <label for="multi-2" class="clr-control-label">Label with for</label>
+              </div>
+              <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+              <span class="clr-subtext">Helper Text</span>
+          </div>
+      </div>
+  
+      <div class="clr-form-control">
+          <label class="clr-control-label">
+              Multi wrappers
+          </label>
+          <div class="clr-control-container">
+              <div class="clr-radio-wrapper">
+                  <input type="radio" id="multi-1" class="clr-radio">
+                  <label for="multi-1">Label with for</label>
+              </div>
+              <div class="clr-radio-wrapper clr-error">
```

Comment:
Same question that I asked for checkboxes for `.clr-error` here. I don't see the error in the demo.

Reply:
Fixed.